tests/test_wave_05.py

```diff
@@ -123,28 +133,22 @@ def test_delete_goal(client, one_goal):
     # Check that the goal was deleted
     response = client.get("/goals/1")
     assert response.status_code == 404
+    assert response_body == { "details": "Goal 1 \"Build a habit of going outside daily\" successfully deleted"}
+    assert Goal.query.get(1) == None
 
-    raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
-
-
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_delete_goal_not_found(client):
-    raise Exception("Complete test")
-
     # Act
-    # ---- Complete Act Here ----
+    response = client.delete("/goals/1")
+    response_body = response.get_json()
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # ---- Complete Assertions Here ----
+    assert response.status_code == 404
+    assert response_body == {'message': 'Goal 1 not found'}
+    assert Goal.query.all() == []
```

Comment:
I would probably forgo this DB check, since the database should have started empty, and it would be pretty surprising if an attempt to read records resulted in records being added. Seeing this here (and for other endpoints that shouldn't generally result in the creation of new records) makes me question whether there's something unexpected going on in the endpoint.